packages/ckeditor5-engine/src/view/placeholder.ts

```diff
@@ -67,6 +70,12 @@ export function enablePlaceholder( { view, element, isDirectHost = true, keepOnF
 
 	if ( element.placeholder ) {
 		setPlaceholder( element.placeholder );
+	} else if ( text ) {
+		setPlaceholder( text );
+	}
+
+	if ( text ) {
```

Comment:
You added a second if statement for the exact same condition that is preceding this condition (`} else if ( text ) {`).

This should be combined into one to be logically consistent.

Reply:
But what if someone has simultaneously an element placeholder's value and passes text as an argument, so the element's value will have higher priority and will be applied but the text is also passed as the argument so we should inform about it, don't we?

Reply:
But what if someone has simultaneously an element placeholder's value and passes text as an argument, so the element's value will have higher priority and will be applied but the text is also passed as the argument so we should inform about it, don't we?

Reply:
Ok sounds fair, a rare case - but indeed we would be on a safe side with this approach 👍